src/System.Drawing.Common/tests/Imaging/MetafileTests.cs

```diff
@@ -481,25 +478,9 @@ public void Ctor_InvalidPathI_ThrowsArgumentException(string fileName)
             using (var bufferGraphics = Graphics.FromHwndInternal(IntPtr.Zero))
             {
                 IntPtr referenceHdc = bufferGraphics.GetHdc();
-                AssertExtensions.Throws<ArgumentException>(null, () => new Metafile(fileName, referenceHdc));
-                AssertExtensions.Throws<ArgumentException>(null, () => new Metafile(fileName, referenceHdc, EmfType.EmfOnly));
-                AssertExtensions.Throws<ArgumentException>(null, () => new Metafile(fileName, referenceHdc, EmfType.EmfOnly, "description"));
-            }
-        }
-
-        [ActiveIssue(22723)]
-        [ActiveIssue(20884, TestPlatforms.AnyUnix)]
-        [ConditionalFact(Helpers.GdiplusIsAvailable)]
-        public void Ctor_PathTooLong_ThrowsPathTooLongException()
-        {
-            string fileName = GetPath(new string('a', 261));
-            using (var bufferGraphics = Graphics.FromHwndInternal(IntPtr.Zero))
```

Comment:
Here also if it makes sense we can keep it if have string `fileName = GetPath(new string('a', shrot.MaxValue));`.